UnitedIncome/components
# [APP-4909] InterstitialScreen Component

### Description
> Please provide a brief description of your changes below.

Implements the `InterstitialScreen` component.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Take a look at it and read through the docs. It's modular enough to be able to be used in multiple different places.

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4909)
